sentry_sdk/serializer.py

```diff
@@ -48,6 +56,17 @@
     # Bytes are technically not strings in Python 3, but we can serialize them
     serializable_str_types = (str, bytes)
 
+
+# Maximum length of JSON-serialized event payloads that can be safely sent
+# before the server may reject the event due to its size. This is not intended
+# to reflect actual values defined server-side, but rather only be an upper
+# bound for events sent by the SDK.
+#
+# Can be overwritten if wanting to send more bytes, e.g. with a custom server.
+# When changing this, keep in mind that events may be a little bit larger than
+# this value due to attached metadata, so keep the number conservative.
+MAX_EVENT_BYTES = 10 ** 6
```

Comment:
Can we clarify what actually happens on the server: That the 200kb limit is applied on teh compressed payload etc. (I get that this is not supposed to reflect that)

Reply:
https://develop.sentry.dev/sdk/envelopes/#size-limits

The 200KB limit is for `/store/`, IIUC `/envelope/` is limited to 20MB compressed, 1MB decompressed (for error events and transactions). So the bottleneck is 1MB decompressed.

I'd be fine lowering the number here if you think we need to be even more conservative.

Would you have a more specific suggestion, what to add here?

Reply:
forgot about envelopes, ignore